src/collaboration_tools/collaboration_tools.md

```diff
@@ -0,0 +1,5 @@
+<h1> ThreeFold's Collaboration Tools </h1>
+
+<h2>Table of Contents</h2>
+
+- [Circle Tool](../circle_tool.md)
```

Comment:
This link doesn't work.
Write this instead:

- [Circle Tool](./circle_tool.md)

(only one dot)